hail/src/main/scala/is/hail/variant/Locus.scala

```diff
@@ -50,17 +50,19 @@ object Locus {
     Locus(contig, elts(size - 1).toInt, rg)
   }
 
-  def parseInterval(str: String, rg: RGBase): Interval = Parser.parseLocusInterval(str, rg)
+  def parseInterval(str: String, rg: RGBase, invalidMissing: Boolean = false): Interval =
+    Parser.parseLocusInterval(str, rg, invalidMissing)
 
-  def parseIntervals(arr: Array[String], rg: RGBase): Array[Interval] = arr.map(parseInterval(_, rg))
+  def parseIntervals(arr: Array[String], rg: RGBase, invalidMissing: Boolean = false): Array[Interval] = arr.map(parseInterval(_, rg, invalidMissing))
```

Comment:
@lawremi I saw you recently changed this same function. I wanted to give you a heads up that this PR adds a new parameter that causes the interval to be set to missing if it doesn't conform to the ReferenceGenome instead of throwing an error. I set the default to false here to preserve the original interface.